client/src/components/MagicalFactor/keywords.js

```diff
@@ -0,0 +1,22 @@
+const keywordsData = {
```

Comment:
would rename it to `KeywordsEnum` (plus, as already mentioned, would like to see the `Keywords` name changed as well). In line with the current `MagicalFactor` name, it could be `MagicalFactorsEnum` or `CommonFactorsEnum`.